rest/example/launchpad/README.md

```diff
@@ -0,0 +1,37 @@
+
+Users of the Launchpad product will need to pass in certain headers in order to make API requests.
```

Comment:
Can you add headers to this? e.g. it should start with something like `# Launchpad`. I kinda disagree with moving this to a separate readme because now it's significantly harder for users to find it.

Reply:
Never mind, I guess the link from the main readme solves that

Reply:
Yeah I think you make a decent case. Katie's point of moving it was to make it easier to link from an external site, so if you think that can be solved while keeping them in the same README then please lmk

Reply:
nah, i think this is okay as is but i still recommend adding a header at the top